src/input.css

```diff
@@ -73,6 +77,10 @@
   border-left: 1px solid rgba(112, 59, 247, 1);
 }
 
+.border-left {
```

Comment:
You are insane? why not add tis tailwind config, and it will be a lot much easy, because you are using same thing for border-star?

Reply:
i use all border for star but some places i just need to use border left or border right therefore i add it